spec/features/user_removes_voter_from_list_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+feature 'user removes a voter from a list', %(
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.